tests/Action/SetObjectFieldValueActionTest.php

```diff
@@ -173,53 +173,6 @@ public function testSetObjectFieldValueActionWithDate(): void
         $this->assertSame(Response::HTTP_OK, $response->getStatusCode());
     }
 
-    public function testSetObjectFieldValueActionWithDateTime(): void
```

Comment:
Why remove this test?

Reply:
See the PR description

Reply:
This test is for features that were mistakenly added #6071